app/services/updated_family_precheck_status_service.rb

```diff
@@ -10,14 +10,18 @@ def call
   private
 
   def pending_approval
-    PrecheckMailer.confirm_charges(family).deliver_now
+    PrecheckMailer.confirm_charges(family).deliver_now unless inelegible_for_precheck?
   end
 
   def changes_requested
-    PrecheckMailer.changes_requested(family, message).deliver_now
+    PrecheckMailer.changes_requested(family, message).deliver_now unless inelegible_for_precheck?
   end
 
   def approved
     family.check_in!
   end
+
+  def inelegible_for_precheck?
```

Comment:
I found this method name a bit confusing because the method is not actually checking whether the family is fully eligible, instead it's just checking two of many requirements.

What do you think of mirroring the method it calls by naming this method `too_late_or_checked_in?`?